pkg/limayaml/default.yaml

```diff
@@ -196,6 +196,11 @@ networks:
 #     hostPortRange: [1, 65535]
 #   # Any port still not matched by a rule will not be forwarded (ignored)
 
+# Message. Information to be shown to the user, given as a Go template for the instance.
+# The same template variables as for the socket above can be used, for instance {{.Dir}}.
```

Comment:
Technically it is the same list of fields as shown by `limactl ls --list-fields`. The fields available in the socket path are just a subset of those.